source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql

```diff
@@ -351,6 +356,7 @@ BEGIN TRY
 		,mdr.ExposureToKnownCaseStatus							AS MDRExposureToKnownCase
 		,mdr.RelationshipToCase									AS MDRRelationshipToCase
 		,mdr.RelatedNotificationId								AS MDRRelatedNotificationId
+		,dbo.ufnYesNo(mdr.DiscussedAtMDRForum)					AS MDRDiscussedAtForum
```

Comment:
Just 1 question @aptemus , do we want Yes/No/NULL or Yes/No/Unknown

Reply:
@LeoCie as the application only supports Yes and No, I think we could just leave this empty if unpopulated